### ISSUE TITLE
Accessibility: focusTrap on p-dialog doesn't work with screen readers

### ISSUE THREAD

**Issue by jennileerose** [primefaces/primeng#8188]
**I'm submitting a ...**  (check one with "x")
```
[X ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
Please demonstrate your case at stackblitz by using the issue template below. Issues without a test case have much less possibility to be reviewed in detail and assisted.

https://stackblitz.com/github/primefaces/primeng-issue-template

**Current behavior**
When in an open p-dialog focus is not trapped when navigating with a screen reader.

**Expected behavior**
When in an open p-dialog focus would not leave the dialog when navigating with a screen reader until close button is activated.

**Minimal reproduction of the problem with instructions**
1. Turn on screen reader (I am using MacOS Voiceover on a Macbook)
2. Open a p-dialog with a form in it via a button-click
3. Navigate through form and once outside the form you leave the dialog.
(I'm working on a project for a private business so I cannot post example code)

**What is the motivation / use case for changing the behavior?**
Making p-dialog work consistently for accessibility reasons.

**Please tell us about your environment:**
MacOS Version 10.14.6 using MacOS Voiceover, in Google Chrome Version 77.0.3865.90. IDE is Webstorm.

* **Angular version:** 
8

* **PrimeNG version:** 
8

* **Browser:**
Google Chrome Version 77.0.3865.90
 
* **Language:** Typescript

* **Node (for AoT issues):** `node --version` =  10 I think

**Comment by cagataycivici** [comment]
Unable to replicate with PrimeNG 9.0.3, if the issue persists please create a new ticket with a test case reproducing the issue e.g. stackblitz or a github repo and it will be reviewed by our team once again.
